modules/core/src/main/scala/org/tesselation/config/types.scala

```diff
@@ -4,17 +4,22 @@ import scala.concurrent.duration.FiniteDuration
 
 import ciris.Secret
 import com.comcast.ip4s.{Host, Port}
+import eu.timepit.refined.types.string.NonEmptyString
 
 object types {
 
   case class AppConfig(
     environment: AppEnvironment,
     keyConfig: KeyConfig,
-    httpClientConfig: HttpClientConfig,
-    externalIp: Host,
-    publicHttp: HttpServerConfig,
-    p2pHttp: HttpServerConfig,
-    cliHttp: HttpServerConfig
+    httpConfig: HttpConfig,
+    dbConfig: DBConfig
+  )
+
+  case class DBConfig(
+    driver: NonEmptyString, // TODO: Make it Regex refined type?
```

Comment:
I'm in favor of not putting TODOs into the `develop` or `master` branches. I think that if something is important to do then it should be either done as a part of the current change or a task on JIRA should be created for it.

Reply:
+1